types/vault.go

```diff
@@ -13,7 +13,8 @@ import (
 )
 
 const (
-	ZeroInterestRate = "0.0"
+	ZeroInterestRate   = "0.0"
+	MaxWithdrawalDelay = 31536000 * 2
```

Comment:
should have a comment here indicating what this duration is.

Reply:
Done.